app/helpers/samples_helper.rb

```diff
@@ -545,14 +545,20 @@ def filter_by_visibility(samples, visibility)
   def filter_by_metadata_key(samples, key, query)
     # TODO(tiago): replace 'filter_by_metadatum' once we decide to includeing not set values
     metadata_field = Metadatum.where(key: key).first
-    samples
-      .includes(metadata: :metadata_field)
-      .where(
-        metadata: {
-          metadata_field_id: metadata_field.metadata_field_id,
-          metadata_field.validated_field => query
-        }
-      )
+
+    samples_with_metadata = samples
+                            .includes(metadata: :metadata_field)
+                            .where(metadata: { metadata_field_id: metadata_field.metadata_field_id })
```

Comment:
This is not from your diff, but... interesting choice to have a column `metadata_field_id` in the table `metadata_fields`... curious what the rationale was for doing that instead of using normal auto-incrementing id. Maybe @MarkAZhang can educate me

Reply:
There actually isn't a metadata_fields.metadata_field_id in the schema but there is a metadata.metadata_field_id from the association. is this line working?

Reply:
https://github.com/chanzuckerberg/idseq-web/blob/master/db/schema.rb#L119

Reply:
oh I see. It's a join table

Reply:
thx